CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # PopupBridge iOS Release Notes
 
+## unreleased
+* Require Xcode 15.0+ (per [App Store requirements](https://developer.apple.com/news/?id=khzvxn8a))
+
```

Comment:
In PPCP iOS we also specified Swift version as well "Require Xcode 15.0+ and Swift 5.9+"
per Jax's suggestion.